.travis.yml

```diff
@@ -3,6 +3,9 @@ language: cpp
 sudo: required
 dist: trusty
 
+git:
+  depth: 1
```

Comment:
Are we sure we want to do this?  [Travis docs](https://docs.travis-ci.com/user/customizing-the-build#Git-Clone-Depth) has this warning:

> Please note that if you use a depth of 1 and have a queue of jobs, Travis CI won’t build commits that are in the queue when you push a new commit.